### DISCUSSION TITLE
print3D/Fusion360/05.fusion360-body

### DISCUSSION THREAD

**Discussion by giscus** [bigrich-luo/DocsGiscus#18] [General]
# print3D/Fusion360/05.fusion360-body

https://docs.geeksman.com/print3D/Fusion360/05.fusion360-body.html

**Comment by thatgame** [comment]
涉及网络的巨卡~ 科学上网都不行，！快放弃了啊 这软件是不是要黄？要黄还不如趁早学别的，

**Reply by StevenZGJ1** [reply]
网络不好不要怪软件 我不科学上网都没事
